18.04/ubuntu-debug/Dockerfile

```diff
@@ -1,13 +1,15 @@
-FROM quay.io/cybozu/golang:1.18-focal AS builder
+FROM --platform=$BUILDPLATFORM quay.io/cybozu/golang:1.18-focal AS builder
 
 WORKDIR /go/src/github.com/fullstorydev/grpcurl
 ARG GRPCURL_VERSION=1.8.7
+ARG TARGETARCH
 RUN curl -fsSL -o grpcurl.tar.gz "https://github.com/fullstorydev/grpcurl/archive/v${GRPCURL_VERSION}.tar.gz" \
     && tar -x -z --strip-components 1 -f grpcurl.tar.gz \
     && rm -f grpcurl.tar.gz \
-    && CGO_ENABLED=0 go install -ldflags "-X \"main.version=${GRPCURL_VERSION}\"" ./cmd/grpcurl
+    && GOARCH=${TARGETARCH} CGO_ENABLED=0 go build -ldflags "-X \"main.version=${GRPCURL_VERSION}\"" -o /go/bin/grpcurl ./cmd/grpcurl
 
-FROM quay.io/cybozu/ubuntu:latest
+ARG TAG
```

Comment:
It looks like the `ARG` must be written before the first `FROM` or the variable will not be supported.
I think the `ARG TAG` must be written before the first `FROM`.
https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact

Reply:
Sorry, I fixed it.